pkgs/development/libraries/webkitgtk/default.nix

```
@@ -93,6 +108,17 @@ stdenv.mkDerivation rec {
       src = ./fdo-backend-path.patch;
       wpebackend_fdo = libwpe-fdo;
     })
+  ] ++ [
+    # https://bugs.webkit.org/show_bug.cgi?id=232934
```

Please include more detailed comments about what each patch does, otherwise I will remove them in the future when I notice the upstream issue is resolved and it builds on Linux without it:

```suggestion
    # Fix build without OpenGL (important on Darwin)
    # https://bugs.webkit.org/show_bug.cgi?id=232934
```

---

Done.